meadServer/client/src/App.js

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
```
if (this.selected !== '')
```

Generally better to early return:

```
if(this.selected === '')
   return;
```

Than to wrap the whole thing in an extra layer of indentation, especially for long methods